### DISCUSSION TITLE
LocalNotificationSchedule for repeating notifications

### DISCUSSION THREAD

**Discussion by divinetouch** [ionic-team/capacitor#3917] [Q&A]
I am trying to create different local notifications that repeat every day, every month, every week, and every year. It seems like it only work for the first notification and then it never repeats.

example:
`{
        every: 'day',
        at: date.toDate(),
        count: 1
        on: {
          hour: 14,
          minute: 28,
        }
}`

In this case the `at: date.toDate() is 2020-12-08T20:28:40.000Z`

I've been setting the `every` property in the `LocalNotificationSchedule` to `day`, `week`, `month`, etc as the documentation suggested https://capacitorjs.com/docs/apis/local-notifications#local-notifications

Setting  the `repeat` to `true` does nothing other than keep showing the notification again and again after I dismissed the notification.

Also from what i understand `count` is for setting the frequency. For example, `count:2` and `every:day` mean every two days but I could be wrong since the document is completely useless when it comes to explaining how to use those properties. Looking at the code doesn't help either.

Does any one know how to properly set a repeatable notification? and how exactly to use `repeat` and `count`?

Thank You!!

**Comment by namrata-Zimbaroos** [comment]
If you are using repeat you have to set **repeats: true** then it will repeat. For **count** I am also not sure how to use that
